ios/PluginTests/PluginTests.swift

```diff
@@ -94,10 +157,14 @@ class PluginTests: XCTestCase {
             ], success: { (result, call) in
                 let resultValue = result!.data["value"] as? Bool
                 XCTAssertTrue(resultValue ?? false)
-                let maybeValue = KeychainWrapper.standard.string(forKey: key)
-                XCTAssertNil(maybeValue)
-                let maybeValue2 = KeychainWrapper.standard.string(forKey: key + "2")
-                XCTAssertNil(maybeValue2)
+                // dedicated keychain wrapper
+                let dedicatedValue = keychainwrapper.string(forKey: key)
+                XCTAssertNil(dedicatedValue)
+                let dedicatedValue2 = keychainwrapper.string(forKey: key + "2")
+                XCTAssertNil(dedicatedValue2)
+                // standard keychain wrapper should not be modified
```

Comment:
If standard keychain entry would not be removed, get() method would always return old value and there is no way how to remove it. we should probably remove entry from standard keychain as well.

Reply:
good point. now the plugin removes the key from the standard and dedicated chain. it now also migrates keys on get calls.